src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -58,6 +63,10 @@ public PersonCard(Person person, int displayedIndex) {
         email.setText("\uD83D\uDCE7\t" + person.getEmail().value);
         // replace birthday and insurance once implemented.
         // insurance.setText("\uD83D\uDCC4\t" + "something something");
+        healthInsurance.setText(person.getHealthInsurance().hasInsurance ? "yes" : "no");
+        disabilityInsurance.setText(person.getDisabilityInsurance().hasInsurance ? "yes" : "no");
+        criticalIllnessInsurance.setText(person.getCriticalIllnessInsurance().hasInsurance ? "yes" : "no");
+        lifeInsurance.setText(person.getLifeInsurance().hasInsurance ? "yes" : "no");
```

Comment:
Do you think this kind of breaks the abstraction principle since we are accessing the direct field of an object? Perhaps using a getter in each insurance object to return the hasInsurance value might be better? 🤔

Reply:
yes that is true, okay will use a getter!

Reply:
fixed 👍